lib/request.coffee

```diff
@@ -322,4 +306,50 @@ module.exports = getRequest = ({
 	# (or resolve to) a new response, or throw/reject.
 	###
 
+	###*
+	# @summary Refresh token on user request
+	# @function
+	# @public
+	#
+	# @description
+	# This function automatically refresh token on user request.
```

Comment:
NIT: this would be better written as `This function automatically refreshes the authentication token.`